nix/sources.json

```diff
@@ -145,15 +145,15 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "ops-lib": {
-        "branch": "master",
+        "branch": "varnish",
```

Comment:
Can change this back to HEAD of master if https://github.com/input-output-hk/ops-lib/pull/51 is merged before this one.